components/search-page/SearchCollectiveCard.js

```diff
@@ -102,7 +102,7 @@ const SearchCollectiveCard = ({ collective, ...props }) => {
                     </Span>
                     {` `}
                     <Span fontSize="12px" fontWeight={400} color="black.700">
-                      <FormattedMessage defaultMessage="Money Spent" />
+                      <FormattedMessage defaultMessage="Contributed" />
```

Comment:
It's important that we reuse strings, we're reaching our quota on Crowdin

```suggestion
                      <FormattedMessage id="AmountContributed" defaultMessage="Contributed" />
```

Reply:
Thanks. I've corrected this. 👍🏽